csrc/host_ir/host_ir.h

```diff
@@ -115,6 +115,39 @@ class PostOnStream : public Expr {
   }
 };
 
+class LaunchKernel : public Expr {
+ public:
+  using Expr::Expr;
+  LaunchKernel(
+      IrBuilderPasskey passkey,
+      int hic_executor_index, // TODO
```

Comment:
code comment what this means. It's associated with the containing HostIrContainer so isn't obvious to the reader.